packages/p2p/src/components/orders/order-details/order-details.scss

```diff
@@ -279,45 +262,52 @@
     }
 
     &__wrapper {
-        width: 456px;
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
+        background-color: var(--general-main-1);
+        border-radius: 4px;
+        border: 0.8rem solid var(--general-section-1);
```

Comment:
```suggestion
        border: 0.8rem solid var(--general-border);
```

Reply:
usually for border, since its fixed, we dont need to use relative measurement, can use px instead of rem

Reply:
I wasn't aware of that, I thought `1rem` was equal to `10px` so went with `rem` values instead. Where can we see the impact of using `rem` vs `px`?